x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.ts

```diff
@@ -143,11 +148,20 @@ export const buildAlertRoot = (
     alertTimestampOverride
   );
   const alertId = generateAlertId(doc);
+  const alertUrl = getAlertDetailsUrl({
```

Comment:
Should the individual building block alerts for EQL sequences have alert URLs as well?

Reply:
Yep, added them